portlets/src/main/frontend/src/apps/profileStats/components/ConnectionsDrawer.vue

```diff
@@ -269,6 +269,7 @@ export default {
     },
     getConnections(offset, limit) {
       return getUserConnections('', offset, limit >= 0 ? limit : this.limit).then(data => {
+        this.connections.splice(0, this.connections.length);
```

Comment:
```suggestion
        this.connections = [];
```

Reply:
You need to append new retrieved list to the already retrieved data. Is the offset updated on each click ? or are you loading all users on each click ?

Reply:
The getconnections() is called on each open of the drawer.

Reply:
@hakermi if there is no load more , then the call will just return the first 20 connections. it is a bad UX since all Connections drawers let users paginate the list, couldn't we use an existing one instead of maintaining each one alone. WDYT @mzorai @AzmiTouil ?

Reply:
There is a load more but it's not paginated, you need to scroll to see the loaded entries

Reply:
@hakermi could you fix the pagination loading then ?

Reply:
in fact @ahamdi the list of connections drawer was not paginated, it is a load more button + scroll to display the list
IMO we just fix the duplication issue in that task and create a new one (quick win) to add a pagination for the drawer (same as done for comments drawer in AS)
cc @hakermi

Reply:
@mzorai Ok, I will validate this one